README.md

```diff
@@ -16,7 +16,7 @@ Lightweight (169MB) [Crashplan](http://www.crashplan.com) docker container.
 docker run -d \
   --name crashplan \
   -h $HOSTNAME \
-  -e TZ \
+  -e TZ=America/Los_Angeles \
```

Comment:
Do NOT set the value in the docker run command.
Only use `-e TZ` and docker will use the value set in the environment where it will be start